cmd/crc/cmd/oc_env.go

```diff
@@ -27,7 +27,7 @@ var ocEnvCmd = &cobra.Command{
 		if err != nil {
 			errors.Exit(1)
 		}
-		output.Outln(shell.GetPathEnvString(userShell, constants.CrcBinDir))
+		output.Outln(shell.GetPathEnvString(userShell, constants.CrcOcBinDir))
```

Comment:
This is going to conflict with https://github.com/code-ready/crc/pull/1136/commits
I don't know in which order you want to merge these? The commits up to `Use Cache interface to cache oc and podman binary` in that PR could be finished and merged without the hosts changes if you prefer to have the refactor first.

Reply:
@cfergeau right, I rebased the #1136 and hope that will merge soon so I can update this one.